.github/workflows/connect-integration.yaml

```diff
@@ -0,0 +1,57 @@
+name: Connect Integration Tests
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
+
+jobs:
+  connect-integration:
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - "preview"   # Nightly builds of Connect
+          - "release"   # special value that always points to the latest Connect release
+          - "2025.09.0" # jammy
+          - "2025.03.0" # jammy (every 6 months, just bc every release would be overkill)
+          - "2024.09.0" # jammy
+          # These versions use R 4.2 and seem to take forever to install R dependencies
```

Comment:
FTR these did eventually pass, but probably not worth 20 minutes of CI time for things that don't change. If things pass on 2023.03.0, they should pass on these versions too.

https://github.com/rstudio/rsconnect/actions/runs/20179566092/job/57936324677?pr=1274